### PR TITLE
Commerce 6570 cdn feature (BOSCH 3)

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-6570

Schermata 2021-05-10 alle 18 02 55

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:stop

**Comment by andrea-ale-sbarra** [comment]
ci:forward

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPAttachmentFileEntryMVCActionCommand.java [hunk 1]
```diff
@@ -188,6 +189,10 @@ protected void updateCPAttachmentFileEntry(ActionRequest actionRequest)
 		double priority = ParamUtil.getDouble(actionRequest, "priority");
 		int type = ParamUtil.getInteger(actionRequest, "type");
 
+		String cdnUrl = ParamUtil.getString(actionRequest, "cdnUrl");
+
+		boolean cdn = Validator.isNotNull(cdnUrl);
```
Comment (Commando18) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPAttachmentFileEntryMVCActionCommand.java:194]: No need to create a variable for this

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v3_3_0/CPAttachmentFileEntryCDNUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * @author Andrea Sbarra
+ */
+public class CPAttachmentFileEntryCDNUpgradeProcess
+	extends BaseCommerceProductServiceUpgradeProcess {
+
+	@Override
+	public void doUpgrade() throws Exception {
+		addColumn(
+			CPAttachmentFileEntryModelImpl.class,
```
Comment (Commando18) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v3_3_0/CPAttachmentFileEntryCDNUpgradeProcess.java:29]: Use Table class instead

**Review by Commando18 — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:reevaluate:1087401_7442

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:reevaluate:1257401_2514

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**